components/dependency_injection/tags.rst

```diff
@@ -133,14 +133,12 @@ custom tag::
     {
         public function process(ContainerBuilder $container)
         {
-            if (!$container->hasDefinition('acme_mailer.transport_chain')) {
+            try {
+                $definition = $container->findDefinition('acme_mailer.transport_chain');
+            } catch (\InvalidArgumentException $e) {
```

Comment:
`has()` / `get()` methods should be used instead yeah. But anyhow, I think this shouldn't be left like this